### ISSUE TITLE
Releases Renamed from Perfect Name to Wrong Name

### ISSUE THREAD

**Issue by Enverex** [nZEDb/nZEDb#1674]
I've seen a lot of these happening so I thought I should create an Issue for it as it's rather detrimental.

I'm seeing lots of releases being renamed from what is literally a perfect name to something either less perfect or at worse, nonsense. An example I just saw was:

    [ TOWN ]-[ www.town.ag ]-[ partner of www.ssl-news.info ]-[ OPEN ] [09/12] - "Pokemon.S17E38.Summer.of.Discovery.480p.HDTV.x264-mSD.part7.rar" - 88,14 MB yEnc

Which originally had the name:

    Pokemon.S17E38.Summer.of.Discovery.480p.HDTV.x264-mSD

Which is exactly correct, BUT, "NFO, nfoChecl: Generic TV 2" just renamed it to:

    Pokemon.S17E38.Summer.of.Discovery.720p.HDTV.x264-W4F : 389MB

nzb_guid for this release is 082cdf41026a560b9a5c377262d72303 if that's useful to look into this. It seems it picked up the (bad) name from the NFO file, where there is a line that states:

    Source : Pokemon.S17E38.Summer.of.Discovery.720p.HDTV.x264-W4F : 389MB

**Comment by Enverex** [comment]
Another. File:

    The.Black.Adder.2.1986.HUN.DVDRip.XviD [09/24] - "the.black.adder.2.1986.hun.dvdrip.xvid.part08.rar" yEnc

Old Name:

    The.Black.Adder.2.1986.HUN.DVDRip.XviD

New name:

    The.Black.Adder.Back&Forth.1999.ENG.DVDRip

Renamed by "NFO, nfoCheck: Generic Movies 2".

**Comment by rainje** [comment]
Some more : 

![2](https://cloud.githubusercontent.com/assets/1025666/7842248/5af79b20-04a2-11e5-93b0-bdf9e38325d1.png)

![3](https://cloud.githubusercontent.com/assets/1025666/7842249/5d76e28e-04a2-11e5-956f-d38f0fc99271.png)

One space before the good release name
![4](https://cloud.githubusercontent.com/assets/1025666/7842254/5f4056ea-04a2-11e5-9383-5635e22730a2.png)

![1](https://cloud.githubusercontent.com/assets/1025666/7842247/579268ca-04a2-11e5-9cfb-743ec47ce1bd.png)


And this one but the old name wasn't good

![5](https://cloud.githubusercontent.com/assets/1025666/7842263/66579a06-04a2-11e5-9e75-53c72da0bbb1.png)

**Comment by rainje** [comment]
Some more...

![wryyy2](https://cloud.githubusercontent.com/assets/1025666/7995724/212ae712-0b19-11e5-9dbb-66a990041b77.png)
![wryyy](https://cloud.githubusercontent.com/assets/1025666/7995723/212ad92a-0b19-11e5-8bb7-2d23cbbe9626.png)
![wryy22](https://cloud.githubusercontent.com/assets/1025666/7995725/212ba116-0b19-11e5-90d7-cc5af070ed34.png)

**Comment by Fossil01** [comment]
I see this ALOT as well. Does it eventually use the name which is shown after "Use Name" ?

**Comment by ruhllatio** [comment]
@kevinlekiller @Bart39 I've been looking into this.  Within NameFixer we have a PreDB format regex as a constant.  I was thinking that if we evaluate the release searchname against this same regex and it was in fact in the correct format already (just no Pre match or else it wouldn't even be in queue) that we could skip the rename section and set it to proc_nfo = 1 thereby removing it from the queue and preventing a completely valid release from being stepped on.  Thoughts?

**Comment by DejaVu** [comment]
Think this needs re-addressing IMO. 
This has been happening today too - 

> New name:     The.Real.McCoys.S03.DVDRip.XviD-OSiTV
Old name:     The.Real.McCoys.S03E15.DVDRip.XviD-OSiTV
New category: Movies SD
Old category: 
Group:        alt.binaries.teevee
Release ID:   5424749
Method:       RequestID->updateRelease<web>

**Comment by Wally73** [comment]
yeah it happens on those complete season releases (it sucks but ok :P )

**Comment by justn6** [comment]
I'm being slammed with these. Also; NZBget will not rename the downloaded files correctly if you're running on windows due to character limits. Any fix for this?
